metacat-functional-tests/metacat-test-cluster/docker-compose.yml

```diff
@@ -50,6 +50,7 @@ services:
                 -Dnetflix.appinfo.metadata.enableRoute53=false
                 -Dmetacat.plugin.config.location=/etc/metacat/catalog
                 -Dmetacat.elacticsearch.enabled=false
+                -Dspring.profiles.active=test
```

Comment:
If you're doing this to force the mysql-metadata to be on for the tests I would say after you rename the profile be sure to enable that feature here.